components/esp_modem/examples/simple_cmux_client/main/Kconfig.projbuild

```
@@ -66,5 +66,11 @@ menu "Example Configuration"
         default n
         help
             Close the multiplexed mode at the end of the example and rollback to command mode.
+            
+    config BROKER_URI
+        string "Broker URL"
+        default "mqtts://mqtt.eclipseprojects.io:8883"
```

Just for the simplicity
```suggestion
        default "mqtt://mqtt.eclipseprojects.io"
```

(so we don't have to add a certificate and don't want to demonstrate an SSL connection with no verification)